source/background.ts

```diff
@@ -42,3 +42,10 @@ browser.runtime.onInstalled.addListener(async ({reason}) => {
 
 // GitHub Enterprise support
 addDomainPermissionToggle();
+
+// `background` fetch required to avoid avoid CORB introduced in Chrome 73 https://chromestatus.com/feature/5629709824032768
+browser.runtime.onMessage.addListener(({action, payload}): Promise<Response> | void => {
+	if (action === 'fetch') {
+		return Promise.resolve(fetch(payload).then(async response => response.json()));
```

Comment:
This is already a promise. Also generally avoid `.then`
```suggestion
		const response = await fetch(payload);
		return response.json();
```

Reply:
Could also test what happens in case of errors in Chrome? Are they correctly passed back to the content script? Code to test:

```js
const response = await fetch('http://doesn.t.exist');
```

and 

```js
const response = await fetch('https://google.com');
```

which exists, but should fail in `.json()`

Reply:
This

Reply:
I cant do `await` without `async` and that we cant do as we said below

Reply:
Darn it. Alright 😃

Reply:
5 Points for trying 😉